wp-content/themes/currentorg/less/current-ltw-projects.less

```diff
@@ -117,27 +162,35 @@
 			margin: 0;
 			vertical-align: initial;
 		}
+		summary::-webkit-details-marker {
+			display: none;
+		}
 	}
 }
 .projects-list {
-	border: 1px solid @grey-d;
-	padding: @common-whitespace * 0.5;
 }
 .projects-list-item {
-	border: 1px solid @grey-d;
+	border: 1px solid @grey-d!important;
 	padding: @common-whitespace * 0.5;
+	background: @white;
 	& + .projects-list-item {
-		margin-top: @common-whitespace * 0.5;
+		margin-top: 0.5rem;
+	}
+	&.active {
+		border-left: 6px solid @azure!important;
 	}
 }
 .projects-single-layout {
 	border: 1px solid @grey-d;
+	border-left: 6px solid @azure;
 	padding: @common-whitespace * 0.5;
-	width: 95%;
+	width: 100%;
+	background: @white;
 	&.open {
 		position: absolute;
-		background: #fff;
+		background: @white;
 		height: 100%;
+		width: 94%;
 	}
 	.projects-single-interior-container {
```

Comment:
![Screen Shot 2020-06-04 at 18 13 14](https://user-images.githubusercontent.com/1754187/83815904-33558c00-a68f-11ea-9b85-e03572b688f7.png)

If the container is taller than the viewport, scrolling down to see content in the container will instead scroll the page, requiring the user to scroll all the way to the bottom of the page before the stuck container stops scrolling. 

A potential solution: 
- `max-height: 99vh` in the general case
- `max-height: calc( 100vh - 56px)` when `.sticky-show & `
- `overflow-y: scroll`

Reply:
Good idea @benlk. I think that solution works well, except in https://github.com/INN/umbrella-currentorg/pull/144/commits/807d841fb7048d489052586dddfe094e6de5d826 I used `95vh` instead to account for the bottom "wallit" area

Reply:
Peculiarities of the modal at 454px wide viewport:

![Screen Shot 2020-06-04 at 19 25 57](https://user-images.githubusercontent.com/1754187/83819946-479e8680-a699-11ea-98a8-6a95fbdb3669.png)

- when scrolled to top, form elements appear atop modal
- modal not full width of column
- close button doesn't match other button styles
- search input not full width

Reply:
@benlk Those should be solved in https://github.com/INN/umbrella-currentorg/pull/144/commits/f70ffdce2564d2fe86ab513bfaaaff9be818a3f0 and https://github.com/INN/umbrella-currentorg/pull/144/commits/34da73ab5e80c574067680f26f0211cce40b8dc0. 

![Screen Shot 2020-06-05 at 9 36 19 AM](https://user-images.githubusercontent.com/18353636/83882385-05fcf280-a710-11ea-8920-c438e40e5b9c.png)

Reply:
https://github.com/INN/umbrella-currentorg/pull/144/commits/ab682187a835663f4883f69cc5e848357f644790 styles the close button to match the search button and also removes the active border from the list item when the modal is closed